axelrod/strategies/titfortat.py

```diff
@@ -790,3 +790,51 @@ def strategy(self, opponent: Player) -> Action:
                 return D
 
         return opponent.history[-1]
+
+
+class RandomTitForTat(Player):
+    """
+    A player starts random, then copies its opponent, then switches
```

Comment:
starts randomly.

You might also consider having it not start randomly, since first round defects trigger some players.

Reply:
Good to know. Will fix that